integration_tests/pages/bookingJourney/selectVisitors.ts

```diff
@@ -0,0 +1,29 @@
+import Page, { PageElement } from '../page'
+
+export default class SelectVisitorsPage extends Page {
+  constructor() {
+    super('Who is going on the visit?')
+  }
+
+  // Visitor totals / age
+  visitorsMaxTotal = (): PageElement => cy.get('[data-test=visitors-max-total]')
+
+  prisonName = (): PageElement => cy.get('[data-test=prison-name]')
+
+  visitorsMaxAdults = (): PageElement => cy.get('[data-test=visitors-max-adults]')
+
+  visitorsMaxChild = (): PageElement => cy.get('[data-test=visitors-max-child]')
```

Comment:
A **very** picky one just for naming, `max-child` maybe `max-children` if we're using the plural of adults 😛